js/app/demo/rulesConfig.jsx

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+define(['react'], function(React) {
+    return React.createClass({
+        displayName: 'RulesConfig',
+        getInitialState: function() {
+            return this.props.config;
+        },
+        buildRules: function(state) {
```

Comment:
hit: would be more canonically called `this.renderRules()` and it should already have access to the state, so may not be needed to pass it in......